api/src/paths/permit-no-sampling.ts

```diff
@@ -104,7 +105,7 @@ export function createNoSamplePermits(): RequestHandler {
 
 export const insertNoSamplePermit = async (
   permit: IPostPermitNoSampling,
-  coordinator: PostCoordinatorData,
+  coordinator: PostCoordinatorData | PutCoordinatorData,
```

Comment:
what'a happening here?

Reply:
this function is now accepting a put or post object for the coordinator data cause we need to insert using this function from both the create and update project permit